boxoffice/static/js/models/sidebar.js

```diff
@@ -0,0 +1,30 @@
+export const SideBarModel = {
+  getItems: function(org, ic) {
+    var sidebar_items = [
```

Comment:
By default, on the home page, there's no sidebar by design. So, this should be an empty array at first.